docs/src/app/pages/legoev3intro.md

```diff
@@ -0,0 +1,25 @@
+# Platform: LEGO® MINDSTORMS® EV3 robot platform
+
+<a href="https://www.lego.com/en-us/mindstorms/about-ev3" target="_blank">Lego Mindstorms EV3</a> is the third generation robotics kit in Lego's Mindstorms product line. It can connect up to four motors and four sensors and has a 100 x 64 pixel LED, audio output. 
+With App Inventor's LegoEv3 components, you can interact Lego EV3 robots easily.
+
+<div style="text-align: center; font-size: 75%; margin: 16pt 0;">
+![Image of the Arduino/Genuino 101 board. Source: Lego.com](/assets/howtos/LegoEV3.png)
+<br>
+Image source: <a href="https://www.lego.com/en-us/mindstorms/about-ev3" target="_blank">Lego EV3 electrical components on Lego.com</a>
+</div>
+
+MIT App Inventor provides complete support to interact with Lego EV3, including:
+<ul><li><a href='http://ai2.appinventor.mit.edu/reference/components/legomindstorms.html#Ev3Motors' target='_blank'>Ev3Motors</a> &mdash; control motor speed and direction.</li></ul>
+<ul><li><a href='http://ai2.appinventor.mit.edu/reference/components/legomindstorms.html#Ev3Ev3ColorSensor' target='_blank'>Ev3ColorSensor</a> &mdash; detect light intensity and color code.</li></ul>
+<ul><li><a href='http://ai2.appinventor.mit.edu/reference/components/legomindstorms.html#Ev3GyroSensor' target='_blank'>Ev3GyroSensor</a> &mdash; detect angular velocity.</li></ul>
+<ul><li><a href='http://ai2.appinventor.mit.edu/reference/components/legomindstorms.html#Ev3TouchSensor' target='_blank'>Ev3TouchSensor</a> &mdash; detect whether this sensor is pressed/released.</li></ul>
+<ul><li><a href='http://ai2.appinventor.mit.edu/reference/components/legomindstorms.html#Ev3UltrasonicSensor' target='_blank'>Ev3UltrasonicSensor</a> &mdash; detect distance between object ahead.</li></ul>
+<ul><li><a href='http://ai2.appinventor.mit.edu/reference/components/legomindstorms.html#Ev3UI' target='_blank'>Ev3UI</a> &mdash; diplay text/number/image on Ev3's 100x64 pixel monochrome LED.</li></ul>
+<ul><li><a href='http://ai2.appinventor.mit.edu/reference/components/legomindstorms.html#Ev3Sound' target='_blank'>Ev3Sound</a> &mdash; play music or short sound effects on Ev3 brick.</li></ul>
+<ul><li><a href='http://ai2.appinventor.mit.edu/reference/components/legomindstorms.html#Ev3Commands' target='_blank'>Ev3Commands</a> &mdash; low level commands can talk to Ev3 brick directly.</li></ul>
+
+Please check Lego EV3 tutorials:
+<ul><li><a href='http://iot.appinventor.mit.edu/assets/howtos/MIT_App_Inventor_EV3_ButtonControl.pdf' target='_blank'>Button Control</a> &mdash; Control Lego EV3 robot with buttons. You can also adjust the motor speed by sliders.</li></ul>
```

Comment:
See my previous note about relative vs absolute URLs and about `<ul>`, `</ul>`, and `<li>`.